rampwf/utils/command_line.py

```diff
@@ -50,10 +50,15 @@ def create_ramp_test_submission_parser():
     parser.add_argument('--pickle', dest='pickle', action='store_true',
                         help='Specify this flag to pickle the submission '
                              'after training.')
-    parser.add_argument('--save-y-preds', dest='save_y_preds',
+    parser.add_argument('--save-output', dest='save_output',
```

Comment:
For consistency with the other parameter, it should be `--save_ouptut` actually.

However, I think that we should refactor using `click` and all parameters will used `-` instead of `_`.
It might be worth to open an issue for `click` refactoring.

This is actually breaking the backend worker as well.